core/modules/user/user.module

```diff
@@ -1003,6 +1006,19 @@ function user_account_form(&$form, &$form_state) {
   // items displayed here.
 }
 
+/**
+ * Returns text to be used as description for whichever role has been set as
+ * admin role.
+ *
+ * @return string
+ *
+ * @see user_admin_roles()
+ * @see user_account_form()
+ */
+function _user_admin_admin_role_description() {
```

Comment:
In my PR, I named this function `_user_admin_admin_...()` because other functions in the `user.admin.inc` file have their names start with `user_admin_*()` or `_user_admin_*()`. Since we moved the function to `user.module instead, then it should be called just `_user_admin_role_description()`:

```suggestion
function _user_admin_role_description() {
```